uqcsbot/scripts/advent.py

```diff
@@ -79,15 +79,14 @@ def from_member_data(cls, data: Dict, year: int, day: Optional[int] = None) -> '
         Times and delta are calculated for the given year and day.
         """
 
-        member = cls(data['name'], data['local_score'], data['stars'])
+        member = cls(data['name'], data['local'], data['stars'], data['global'])
```

Comment:
These dict keys are from the API's JSON. Are you sure these should/can be changed?

Reply:
They should not.